### ISSUE TITLE
No batch size? No normalization?

### ISSUE THREAD

**Issue by mb-Ma** [Aminsheykh98/INCREASE-pytorch#1]

**Comment by mb-Ma** [comment]
I see. The normalization is located in the forward function.

**Comment by Reset-quick** [comment]
Hello, may I ask if you have now solved the problem that the training set and the validation set in the data set METR-LA become 0 after a few epoches? Can you run out the effect in the paper?

**Comment by Aminsheykh98** [comment]
Hello, @Reset-quick 

Unfortunately, I haven't resolved the issue. I've been occupied with other projects recently. However, if I resume work on this, I will keep you updated on any progress.

Best regards,
